packages/cli/build/dev.ts

```diff
@@ -5,8 +5,13 @@ import onMain from './main'
 import { build, dev } from './typescript'
 
 async function main() {
-    await build()
+    // await build()
     dev()
+    if (process.argv[2] === '--daemon') {
```

Comment:
why not `process.argv.include('--daemon')`?

Reply:
this commit has already picked to the master branch because it is blocking the dev workflow. please open a new issue or PR for it